vscode/src/custom-prompts/utils/menu.ts

```diff
@@ -146,3 +112,68 @@ export const promptSizeInit = {
     default: 0,
     'recently used': 0,
 }
+
+// Ask user to confirm before trying to delete the cody.json file
+export async function showRemoveConfirmationInput(): Promise<string | void> {
+    const confirmRemove = await window.showWarningMessage(
+        'Are you sure you want to remove the .vscode/cody.json file from your file system?',
+        { modal: true },
+        'Yes',
+        'No'
+    )
+    return confirmRemove
+}
+
+// Quick pick menu to select a command from the list of available Custom Commands
+
+export async function commandPicker(promptList: string[] = []): Promise<string> {
+    const selectedRecipe = (await window.showQuickPick(promptList)) || ''
+    return selectedRecipe
+}
+
+export async function quickChatInput(): Promise<void> {
+    const humanInput = await window.showInputBox({
+        prompt: 'Ask Cody a question...',
+        placeHolder: 'ex. What is a class in Typescript?',
```

Comment:
Some more consistency :)
 
```suggestion
        placeHolder: 'e.g. What is a class in TypeScript?',
```